activiti-cloud-starter-org/src/main/java/org/activiti/cloud/organization/OrganizationSwaggerConfiguration.java

```diff
@@ -16,27 +16,33 @@
 
 package org.activiti.cloud.organization;
 
+import java.util.function.Predicate;
+
+import org.activiti.cloud.common.swagger.DocketCustomizer;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.RequestHandler;
+import springfox.documentation.builders.ApiInfoBuilder;
 import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @Configuration
-@EnableSwagger2
 public class OrganizationSwaggerConfiguration {
 
     @Bean
     @ConditionalOnMissingBean(Docket.class)
-    public Docket api() {
-        return new Docket(DocumentationType.SWAGGER_2)
-                .select()
-                .apis(RequestHandlerSelectors.basePackage("org.activiti.cloud.services.organization.rest"))
-                .paths(PathSelectors.any())
-                .build();
+    public Predicate<RequestHandler> apiSelector() {
+        return RequestHandlerSelectors.basePackage("org.activiti.cloud.services.organization.rest")::apply;
+    }
+
+    @Bean
+    public DocketCustomizer docketCustomizer(){
+        return (docket) -> docket.apiInfo(new ApiInfoBuilder()
```

Comment:
Can you make the apiInfo a bean (with @ConditionalOnMissingBean), to allow re-defining it in APS? I think, this should be the same for all swagger apiInfos created in Activiti.

Reply:
It makes sense. I'll update It.